src/Test/Monad/Control.hs

```diff
@@ -41,6 +41,10 @@ liftWith_lift
   => m a -> Equation (t m a)
 liftWith_lift m = liftControl @t m :=: lift m
 
+-- |
+-- @
+-- ('liftWith' (\run -> run t) '>>=' 'restoreT' . 'return') = t
```

Comment:
Backslashes need to be escaped, if I recall correctly. (double check the haddocks)